Makefile

```diff
@@ -0,0 +1,25 @@
+docstyle:
+	echo "Checking documentation with pydocstyle..."
+	python -m pydocstyle ${SRC_DIR}/pymc4/
+	echo "Success!"
+
+
+format:
+	echo "Checking code style with black..."
+	python -m black -l 100 --check pymc4/
+	echo "Success!"
+
+
+style:
+	echo "Checking code style with pylint..."
+	python -m pylint pymc4/
+	echo "Success!"
+
+
+test:
+	pytest -v pymc4/tests/ --cov=pymc4/
+
+lint: docstyle format style
+
+
+check: lint test
```

Comment:
Deleting `lint.sh` is going to be break CI. `.travisyml` file will need to be updated as well

Reply:
Yep, exactly what I expected. I have identified the lines that I need to modify:

```
  - if [[ $NAME == UNIT ]]; then docker run  --mount type=bind,source="$(pwd)",target=/opt/pymc4 --name pymc4_test pymc4:latest bash -c \
                                              "pytest -v pymc4/tests/ --cov=pymc4/";
    fi

  - if [[ $NAME == LINT ]]; then docker run pymc4:latest bash -c "./scripts/lint.sh";
    fi
```

How would you modify these? I'm not quite well-versed with Docker.

Reply:
>   - if [[ $NAME == LINT ]]; then docker run pymc4:latest bash -c "make lint";
>     fi

That should work. You can leave the other one for now it'll still function

Reply:
Solved.